CommonConfig.mk

```diff
@@ -105,8 +105,21 @@ ifeq ($(HOST_OS),linux)
 endif
 WITH_DEXPREOPT_BOOT_IMG_AND_SYSTEM_SERVER_ONLY ?= true
 
+# Build kernel using kernel's Android.mk file.
+# May be overriden by KernelConfig.mk if prebuilt kernel present.
+# Can also be turned off in Customization.mk in case it is desired to use a
+# custom ROM's kernel build system, e.g. LineageOS' or PE's.
+BUILD_KERNEL ?= true
+
 -include $(KERNEL_PATH)/common-kernel/KernelConfig.mk
 
+ifeq ($(BUILD_KERNEL),true)
+BOARD_DTBO_IMAGE_NAME := dtbo-$(TARGET_DEVICE).img
+BOARD_PREBUILT_DTBOIMAGE ?= $(PRODUCT_OUT)/$(BOARD_DTBO_IMAGE_NAME)
+else
+BOARD_PREBUILT_DTBOIMAGE := kernel/sony/msm-4.14/common-kernel/dtbo-$(TARGET_DEVICE).img
```

Comment:
Why not use $(BOARD_DTBO_IMAGE_NAME) here as well?

Reply:
We might change `BOARD_DTBO_IMAGE_NAME` to `dtbo-$(TARGET_ARCH)` or sth later, so I kept the two separate.